pcsx2/COP0.cpp

```diff
@@ -465,7 +465,7 @@ void MFC0()
 			[[fallthrough]];
 
 		default:
-			cpuRegs.GPR.r[_Rt_].UD[0] = (s64)cpuRegs.CP0.r[_Rd_];
+			cpuRegs.GPR.r[_Rt_].UD[0] = (s32)cpuRegs.CP0.r[_Rd_];
```

Comment:
Could be `SD[0]` to better indicate to the reader that we're writing a signed value back, but the codegen is the same.

Reply:
Changed it